### DISCUSSION TITLE
有人解释一下RSS路由到底是个什么意思起个什么作用吗。

### DISCUSSION THREAD

**Discussion by Squirre17** [DIYgod/RSSHub#11642] [Q&A] [answered]
目前在互联网上没找到好的解释

**Comment by qizidog** [comment]
理解为可个人定制版的微信公众号

**Comment by HPDell** [comment, marked as answer]
路由是网页开发中的常见名词，简单的说，就是对一个匹配的URL指定什么样的处理。比如，我现在访问 

`https://rsshub.app/github/repos/DIYgod/`

浏览器知道往服务器 `rsshub.app` 发送 https 请求，请求的资源地址是 `/github/repos/DIYgod/` 。服务器收到这个请求，它怎么处理呢？此时服务器找了一下，发现由一个**路由**声称它要处理发到这个地址的请求，然后服务器就让这个路由处理了。根据这个路由指定的处理模块，它就获取了 `DIYgod` 这个用户下面的所有仓库，生成对应的 RSS 页面，让服务器返回给发来请求的浏览器。

建议可以看一下 [express.js 等框架的文档](https://expressjs.com/en/guide/routing.html)。
